cmd/kubeapps-apis/plugins/fluxv2/packages/v1alpha1/harbor_api_v2_repo_lister.go

```diff
@@ -0,0 +1,298 @@
+// Copyright 2021-2022 the Kubeapps contributors.
+// SPDX-License-Identifier: Apache-2.0
+package main
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strings"
+	"unicode"
+
+	httpclient "github.com/vmware-tanzu/kubeapps/pkg/http-client"
+	"helm.sh/helm/v3/pkg/registry"
+	log "k8s.io/klog/v2"
+	"oras.land/oras-go/v2/errdef"
+	orasregistryv2 "oras.land/oras-go/v2/registry"
+	orasregistryauthv2 "oras.land/oras-go/v2/registry/remote/auth"
+)
+
+// This flavor of OCI repsitory ister works with respect to to VMware Harbor
```

Comment:
```suggestion
// This flavor of OCI repository lister works with respect to VMware Harbor
```

Also, I don't think it is just for the VMware harbor instance, no? Ah, perhaps you meant the VMware Harbor product? If so, I think Harbor is not -just-a VMware project, but a Harbor is a [Cloud Native Computing Foundation](https://cncf.io/) one, as it appears as a Graduated project on their website.

Anyway, just being nitpicky, feel free to ignore the comment :P

Reply:
will fix comment